src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/Models/RazorCodeAction.cs

```diff
@@ -33,7 +33,7 @@ internal class RazorCodeAction : CodeAction, IRequest<RazorCodeAction>, IBaseReq
         public new Command Command { get; set; }
 
         [JsonProperty(PropertyName = "data", NullValueHandling = NullValueHandling.Ignore)]
-        public object Data { get; set; }
+        public new object Data { get; set; }
```

Comment:
@TanayParikh ditto

Reply:
This `new` shouldn't be necessary as there isn't a `Data` property in the O# CodeAction which we need to "hide"

The reason we needed the expand to using a RazorCodeAction was to be able to support the `Data` field.

Reply:
Edit: looks like they just added the Data field. This new may be needed now / we may be able to remove the `RazorCodeAction` in its entirety.

Reply:
> src\Razor\src\Microsoft.AspNetCore.Razor.LanguageServer\CodeActions\Models\RazorCodeAction.cs(36,27): error CS0109: (NETCORE_ENGINEERING_TELEMETRY=Build) The member 'RazorCodeAction.Data' does not hide an accessible member. The new keyword is not required.

This change will likely be needed in the next PR where the O# version is upgraded.